pyrealm/splash/solar.py

```diff
@@ -113,78 +118,76 @@ def __post_init__(
         self.dr = np.expand_dims(dr, axis=expand_dims)
         self.delta = np.expand_dims(delta, axis=expand_dims)
 
-        # Calculate variable substitutes (u and v), unitless
-        self.ru = np.sin(np.deg2rad(self.delta)) * np.sin(np.deg2rad(lat))
-        self.rv = np.cos(np.deg2rad(self.delta)) * np.cos(np.deg2rad(lat))
+        self.ru, self.rv = calc_lat_delta_intermediates(self.delta, lat)
 
         # Calculate the sunset hour angle (hs), Eq. 3.22, Stine & Geyer (2001)
-        self.hs = (
-            np.arccos(-1.0 * np.clip(self.ru / self.rv, -1.0, 1.0))
-            / self.core_const.k_pir
-        )
+        self.hs = calc_sunset_hour_angle(self.delta, lat, self.core_const.k_pir)
 
         # Calculate daily extraterrestrial solar radiation (ra_d), J/m^2
         # Eq. 1.10.3, Duffy & Beckman (1993)
-        self.ra_d = (
-            (86400.0 / np.pi)
-            * self.core_const.k_Gsc
-            * self.dr
-            * (
-                self.ru * self.core_const.k_pir * self.hs
-                + self.rv * np.sin(np.deg2rad(self.hs))
-            )
+        self.ra_d = calc_daily_solar_radiation(
+            self.core_const.k_Gsc,
+            self.dr,
+            self.core_const.k_pir,
+            self.hs,
+            self.delta,
+            lat,
```

Comment:
I've a strong preference (that is probably wildly inconsistently applied) for naming arguments to function calls and not relying on argument order.

Reply:
I'll update these functions for sure. Would you like this applied as a general rule?

Reply:
Yeah, I think so.